### ISSUE TITLE
Add a more advanced example

### ISSUE THREAD

**Issue by arvidsson** [arvidsson/bt#21]

**Comment by selimanac** [comment]
Yes please, please :) 
BrainTree looks really great. First of all, thank you for that. 
I'm planning to implement this lib on [Defold](https://www.defold.com/) as native extension. More advanced and working example could be very very helpful to me. This one is [not working](https://github.com/arvidsson/BrainTree/issues/18#issuecomment-377506169) as it is. I couldn't find any documentation about the usage :(

Also I thing I should update/tick repeating decorators from a game loop and  I will need this: https://github.com/arvidsson/BrainTree/issues/19

**Comment by arvidsson** [comment]
Yeah I will create more examples and create more proper documentation for the different nodes you can use in the behavior tree. Thanks for the kind words! :)

**Comment by arvidsson** [comment]
I have added a first simple example, and plan to add more in the future.

**Comment by rudycazabon** [comment]
Greetings, dropping by to say that BrainTree looks interesting ... however, a few more fleshed out samples (beyond the simple case) would be helpful in getting the fullness of what you have in mind with it.
